epyqlib/variableselectionmodel.py

```diff
@@ -600,20 +604,22 @@ def update_parameter(node, cache):
                 return
 
             if test(node):
-                # TODO: CAMPid 0457543543696754329525426
-                chunk = cache.new_chunk(
-                    address=int(node.fields.address, 16),
-                    bytes=self.zero_bytes(node.fields.size),
-                    reference=node,
-                )
-                cache.add(chunk)
-
-                if subscribe:
-                    callback = functools.partial(
-                        self.update_chunk,
-                        node=node,
+                # Check specifically for None value. Zero shouldn't be a value for size, but just in case.
+                if node.fields.size is not None:
+                    # TODO: CAMPid 0457543543696754329525426
```

Comment:
Would be nice to refactor these campids as we come across them. If there's not a huge rush on this ticket relative to the refactor effort for this particular campid

Reply:
I simplified the changes so this area is no longer affected. Since this is a "quick" bug fix in a complex area of the code, I recommend holding off on remedying the CAMPid's.

Reply:
Thanks, guys... I agree with both assessments. I am an advocate for not leaving things with bad smells lingering and addressing them when "in the area", but I also agree with Gordon on not undertaking a potential rabbit hole (as there seem to be many campy things in this code 😄 )